website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -78,6 +78,8 @@ Follow the same process in [Installing dbt Cloud CLI](#manually-install-windows-
 > $ pwd
```

Comment:
```suggestion
a. $ pwd ## Input
```

Reply:
I think it might be worth adding these in a future iteration, but for the current (small!) beta group they haven't been confused by these lines.

Reply:
hey @greg-mckeon , yes i see what you mean and would adding these suggestions now make it easier for future users since the changes will already be in the docs? if the current beta group are clear about these lines, my perspective is that making it explicitly clear won't affect their understanding. basically, there's no harm in adding the suggestions right? -- what do you think?

Reply:
hey @greg-mckeon , what do you think about me removing the `$` from this page? it removes the need to explain what the dollar sign means. also when you click the copy icon on the code block, it copies everything including the `$`.